src/renderers/dom/client/eventPlugins/ChangeEventPlugin.js

```diff
@@ -142,7 +152,7 @@ if (ExecutionEnvironment.canUseDOM) {
   // IE9 claims to support the input event but fails to trigger it when
   // deleting text, so we ignore its input events
   isInputEventSupported = isEventSupported('input') && (
-    !('documentMode' in document) || document.documentMode > 9
+    !('documentMode' in document) || document.documentMode > 11
```

Comment:
If there never is, then maybe there is no harm?  I suppose I'd be fine either way, doesn't really matter to me, hard to tell when future proofing is beneficial/detrimental.  @syranide let me know if you have a strongly preferred solution here.  Maybe @spicyj has a strong feeling on this topic?